tests/ipfwd/test_dir_bcast.py

```diff
@@ -1,19 +1,106 @@
 import pytest
+import json
+import logging
 
 from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory   # lgtm[py/unused-import]
+from tests.common.portstat_utilities import parse_column_positions
 from tests.ptf_runner import ptf_runner
 from datetime import datetime
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_rand_selected_tor_m    # lgtm[py/unused-import]
 
 pytestmark = [
     pytest.mark.topology('t0', 'm0')
 ]
 
-def test_dir_bcast(duthosts, rand_one_dut_hostname, ptfhost, tbinfo):
+logger = logging.getLogger(__name__)
+
+PTF_TEST_PORT_MAP = '/root/ptf_test_port_map.json'
+
+def parse_mux_status(output_lines):
```

Comment:
Can you take advantage of the existing `show_and_parse` method to avoid code duplication: https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/devices/sonic.py#L1387